tools/scripts/rockset_autoscale.py

```diff
@@ -1,10 +1,10 @@
 import rockset
 import os
-import time
+from time import sleep
 from datetime import datetime, time
```

Comment:
time is unused

Reply:
Well, not it's used here (for type annotations)
https://github.com/pytorch/test-infra/blob/23b7238f06f6f7383d4112ed15c2c94faffd09c8/tools/scripts/rockset_autoscale.py#L36